nixos/modules/services/desktop-managers/plasma6.nix

```diff
@@ -170,7 +170,7 @@ in {
         breeze.qt5
         plasma-integration.qt5
         pkgs.plasma5Packages.kwayland-integration
-        pkgs.plasma5Packages.kio
+        (pkgs.plasma5Packages.kio.override { withoutKcms = true; })
```

Comment:
Maybe we should flip the flag here? I think nixpkgs generally tends to prefer `withFoo` options to `withoutFoo`.

Reply:
changed that